PKHeX.WinForms/Controls/SAV Editor/SAVEditor.cs

```diff
@@ -1097,5 +1097,13 @@ private void ClickShowdownExportBattleBox(object sender, EventArgs e)
             catch { }
             WinFormsUtil.Alert("Showdown Team (Battle Box) set to Clipboard.");
         }
+
+        private void B_OpenUGSEditor_Click(object sender, EventArgs e)
+        {
+            if (SAV.Generation!=4 && !SAV.DP || SAV.Generation != 4 && !SAV.Pt)
```

Comment:
if you wanna make it a bit cleaner
```
             switch (SAV.Version)
             {
                 case GameVersion.DP:
                 case GameVersion.Pt:
                     new SAV_Underground(SAV).ShowDialog(); break;
             }
```

Reply:
Oh, I'm going to do it this way, thanks!